### ISSUE TITLE
From Other Suns touchpad functionality is broken

### ISSUE THREAD

**Issue by JewelRunnerVR** [LibreVR/Revive#814]
So i dont know if this has been reported but in From Other Suns using the touchpad is extremely cumbersome for locomotion. Basically in order to move in any direction you must place your thumb at the center of the touchpad and slide it in the direction you want to move. If you at any point lift your thumb then you must realign to the center if your touchpad again. This makes for very horrid locomotion experience in this game and borderline unplayable in many cases. 

Any thoughts on a way to adress this issue?

**Comment by Armada651** [comment]
You don't need to re-align with the center, though you will need to slide every time. I'm currently working on alternative mappings for the 1.2 stable release that should help with this issue.

**Comment by DeutschRitter** [comment]
Yes, locomotion is very bad atm in this game! It makes it very hard!

**Comment by ceaser** [comment]
@JewelRunnerVR, @DeutschRitter, I made a build just so my friends with the Vive can play From Other Suns too. It's a short-term fix until @Armada651 make a build with custom configurations. Another thing I notice with the changes since 1.2 beta2 is the frame rate is also better on my machines.

My fork and build are located at https://github.com/ceaser/Revive/releases

**Comment by Fengar** [comment]
@Armada651 is still working with that fix? I need more hints on how to play this game with revive, I was able to launchm, but my screen went black and i needed to re center something, I pressed both pads as the tutorial says but nothing happened.
